usaon_vta_survey/templates/macros/forms.j2

```diff
@@ -155,3 +155,46 @@
     </ul>
   {% endif %}
 {%- endmacro -%}
+
+
+{% macro user(form) -%}
```

Comment:
Nice! For consistency with the other macros, let's call this `user_fields` instead of `user`